cmd/skopeo/utils.go

```diff
@@ -200,6 +192,22 @@ func (opts *imageDestOptions) newSystemContext() (*types.SystemContext, error) {
 	return ctx, err
 }
 
+func (opts *globalOptions) newSystemContext() *types.SystemContext {
```

Comment:
This should have documentation, at least the “guaranteed to return a fresh instance, so it is safe…” part.